static/app/views/replays/detail/network/details/onboarding.tsx

```diff
@@ -190,17 +190,17 @@ function SetupInstructions({
 
   const title =
     showSnippet === Output.SETUP
-      ? t('Capture Request and Response Headers and Payloads')
+      ? t('Capture Request and Response Headers and Bodies')
       : visibleTab === 'details'
       ? t('Capture Request and Response Headers')
-      : t('Capture Request and Response Payloads');
+      : t('Capture Request and Response Bodies');
 
   return (
     <StyledInstructions data-test-id="network-setup-steps">
       <h1>{title}</h1>
       <p>
         {tct(
-          `To protect user privacy, Session Replay defaults to not capturing the request or response headers. However, we provide the option to do so, if it’s critical to your debugging process. [link]`,
+          `To protect user privacy, Session Replay defaults to not capturing the request or response headers. However, we provide the option to do so, if it’s critical to your debugging process. [link].`,
```

Comment:
I think we should mention bodies here because this shows up on the request/resp bodies tabs

```suggestion
          `To protect user privacy, Session Replay defaults to not capturing the request or response headers and bodies. However, we provide the option to do so, if it’s critical to your debugging process. [link].`,
```

Reply:
and i missed a period after the link :(